png_benchmark.py

```diff
@@ -0,0 +1,144 @@
+                if frame_index == FRAME_TO_SAVE:
+                    img.save(pathlib.Path(OUTPUT_PATH, f"ct{compress_type}_cl{compress_level}.png"), "PNG", compress_level=compress_level, compress_type=compress_type)
+            
+            # Save average test results
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_TIME_MS] = max_time_ns / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_TIME_MS] = min_time_ns / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_TIME_MS] = total_time_ns / FRAME_COUNT / 1e6
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_SIZE_B] = max_size_B
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_SIZE_B] = min_size_B
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_SIZE_B] = total_size_B / FRAME_COUNT
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MAX_COMPRESSION] = max_compression
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][MIN_COMPRESSION] = min_compression
+            results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"][AVG_COMPRESSION] = total_compression / FRAME_COUNT
+            print(f"Compress level {compress_level} complete")
+    
+    print("")
+    print("-------------------TEST COMPLETED------------------")
+    print("")
+    
+    # Saving full results
+    with open(pathlib.Path(OUTPUT_PATH, "results.json"), 'w', encoding="utf-8") as file:
+        file.write(json.dumps(results, indent=2))
+    
+    # Saving shortcut results without frame data (for more human readability)
+    with open(pathlib.Path(OUTPUT_PATH, "summary.csv"), 'w', encoding="utf-8") as file:
+        file.write("Compression Type,Compression Level,Max Time (ms),Min Time (ms),Avg Time (ms),Max Size (B),Min Size (B),Avg Size (B),Max Size Ratio (%),Min Size Ratio (%),Avg Size Ratio (%)\n")
+        for compress_type in COMPRESS_TYPES:
+            for compress_level in range(INITIAL_COMPRESS_LEVEL, MAX_COMPRESS_LEVEL+1):
+                current_result = results[f"compress_type_{compress_type}"][f"compress_level_{compress_level}"]
+                line = f"{compress_type},{compress_level},{current_result[MAX_TIME_MS]},{current_result[MIN_TIME_MS]},{current_result[AVG_TIME_MS]},{current_result[MAX_SIZE_B]},{current_result[MIN_SIZE_B]},{current_result[AVG_SIZE_B]},{current_result[MAX_COMPRESSION]},{current_result[MIN_COMPRESSION]},{current_result[AVG_COMPRESSION]}\n"
+                file.write(line)
+    
+    test_end = time.time()
+    print("End time:", test_end)
+    print(f"Time taken: {int((test_end-test_begin)/60)} mins {int(test_end-test_begin)%60} secs")
```

Comment:
Add spaces around subtraction operator, division operator, and modulus operator (`%` ).